### DISCUSSION TITLE
Completely turn off the provider for a given URL

### DISCUSSION THREAD

**Discussion by obones** [git-ecosystem/git-credential-manager#1475] [Q&A] [answered]
Hello,

In order to avoid error message in my logs, I am trying to completely turn off for a given URL for which I have defined a helper like this:

```
[credential "https://my.private.server.lan"]
    helper = !my_helper --some-option $@
    useHttpPath = true
    interactive = false
```

I have set `interactive = false` to avoid the prompt but this gives me this kind of messages:

```
fatal: Cannot prompt because user interactivity has been disabled.
```

As such I would like to completely turn off the provider for this URL while keeping using it for all other repositories that I'm using.

Is that possible, and if yes, how to do it?

**Comment by hickford** [comment, marked as answer]
Try

```
[credential "https://my.private.server.lan"]
    helper = 
    helper = !my_helper --some-option $@
```

https://git-scm.com/docs/gitcredentials#_configuration_options

> If credential.helper is configured to the empty string, this resets the helper list to empty (so you may override a helper set by a lower-priority config file by configuring the empty-string helper, followed by whatever set of helpers you would like).

**Reply by obones** [reply]
Thanks, that's exactly it. It wasn't clear to me that one had to "reset" the helper by specifying a blank value then my new value.
And it's now even faster to use, very much appreciated.

**Comment by EimalShirzai** [comment]
Yes
